{"name": "test_failed_login[invalid_user-wrong_pass-Epic sadface: Username and password do not match any user in this service]", "status": "passed", "steps": [{"name": "Test invalid login: invalid_user/wrong_pass", "status": "passed", "start": 1754306251451, "stop": 1754306251784}], "attachments": [{"name": "log", "source": "32da41e3-add4-4c2e-9fc9-f5414f166541-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "9abb8cda-8fb0-4c35-adbd-7ed4211e8396-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "'invalid_user'"}, {"name": "password", "value": "'wrong_pass'"}, {"name": "error_message", "value": "'Epic sadface: Username and password do not match any user in this service'"}], "start": 1754306251451, "stop": 1754306251784, "uuid": "0beba188-a627-4349-9c92-904290178d8a", "historyId": "009e9186f369e24c9a13e6a3a9bf681d", "testCaseId": "eb98303b78b0dd8a81a7d4c9688110a7", "fullName": "tests.test_login#test_failed_login", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "regression"}, {"name": "story", "value": "Failed Login"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "LAPTOP-E3GPSS8Q"}, {"name": "thread", "value": "2904-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}], "titlePath": ["tests", "test_login.py"]}